compute/admin_guide/waas/deploy_waas/deployment_hosts.adoc

```diff
@@ -77,10 +77,11 @@ NOTE: If your application uses *TLS* or *gRPC*, you must specify a port number.
 
 .. Enter *WAAS Port (only required for Windows or when using xref:../waas_advanced_settings.adoc#remote-host["Remote host"] option)*.
 +
+image::waas_endpoint_lineitem_app_embbded.png[width=550]
 Specify the TCP port on which WAAS listens. WAAS receives traffic from your end-users on this port, processes it, and then sends it to your app on the App port.
 +
 NOTE: Protecting Linux-based hosts does not require specifying a *`WAAS port`* since WAAS listens on the same port as the protected application.
-Because of Windows internal traffic routing mechanisms WAAS and the protected application cannot use the same *`App port`*. Consequently, when protecting Windows-based hosts the *`WAAS port`* should be set to the port end-users send requests to, and the *`App port`* should be set to a *different* port on which the protected application would listen on and WAAS would forward traffic to.  
+Because Windows has its own internal traffic routing mechanisms, WAAS and the protected application cannot use the same *`App port`*. Consequently, when protecting Windows-based hosts the *`WAAS port`* should be set to the port end-users send requests to, and the *`App port`* should be set to a *different* port on which the protected application will listen and to which WAAS will forward traffic.
```

Comment:
Thanks for your inputs. This section has been rearranged, and the changes are included in this [PR](https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/757/commits/31e95000c6a48c31b1110ae40e375f518cd24c6f#diff-2b9244012de95d5c1c341aa689d69daadbdd0d9e9ae8d8f7004dff8de2fd4e52).
Cc: @Pubs-MV

Reply:
If you have merged @pan0ptic0n's feedback, can we close this PR @divyabhushan.